pkg/push/api/api.go

```diff
@@ -145,19 +145,10 @@ func (s *PushService) CreatePush(
 		}
 		return nil, dt.Err()
 	}
-	if s.containsFCMKey(ctx, pushes, req.Command.FcmApiKey) {
-		if status.Code(err) == codes.AlreadyExists {
-			dt, err := statusFCMKeyAlreadyExists.WithDetails(&errdetails.LocalizedMessage{
-				Locale:  localizer.GetLocale(),
-				Message: localizer.MustLocalize(locale.AlreadyExistsError),
-			})
-			if err != nil {
-				return nil, statusInternal.Err()
-			}
-			return nil, dt.Err()
-		}
+	if err := s.checkFCMServiceAccount(ctx, pushes, req.Command.FcmServiceAccount, localizer); err != nil {
```

Comment:
This function validates the JSON file and checks if it does not already exist in the DB.